tests/d3d12_crosstest.h

```diff
@@ -296,6 +297,10 @@ static ID3D12Device *create_device(void)
         return NULL;
     }
 
+    /* Enable support for 6_3+, we need this for some tests. */
```

Comment:
6.4+ perhaps. DXR is 6.3, and that seems to work without enabling experimental features.

Reply:
6.3 is fine but 6.3 with SV_ShadingRate being used requires it

Reply:
Ok